### PR TITLE
Update go-mod-bootstrap to v0.0.18

### PR DESCRIPTION
**Pull request by michaelestrin**
Fixes: https://github.com/edgexfoundry/edgex-go/issues/2350
Signed-off-by: Michael Estrin <m.estrin@dell.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/edgexfoundry/edgex-go/issues/2350

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Comment by michaelestrin** [comment]
Rebased

File: internal/core/data/main.go [hunk 1]
```diff
@@ -48,7 +50,7 @@ func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router, re
 	//      ....
 	//      flags.Parse(os.Args[1:])
 	//
-	flags := commandline.NewDefaultCommonFlags("")
+	flags := flags.New()
```
Comment (lenny-intel) [inline internal/core/data/main.go]: In other services you changed `flags` to `f`. Should be consistent.

Comment (michaelestrin) [reply internal/core/data/main.go]: Can't believe I missed that.  Thank you for catching it.

**Comment by lenny-intel** [comment]
@michaelestrin , you are still missing change to internal/security/secrets/main.go

**Comment by michaelestrin** [comment]
> you are still missing change to internal/security/secrets/main.go

Fixed.  @lenny-intel

**Comment by michaelestrin** [comment]
Updated commit hash to force rebuild.  edgex-go-master-verify-arm failed because of network error.

**Pull request merged**